src/SparkFun_u-blox_GNSS_Arduino_Library.h

```diff
@@ -950,6 +950,11 @@ class SFE_UBLOX_GNSS
   bool setDynamicModel(dynModel newDynamicModel = DYN_MODEL_PORTABLE, uint16_t maxWait = defaultMaxWait);
   uint8_t getDynamicModel(uint16_t maxWait = defaultMaxWait); // Get the dynamic model - returns 255 if the sendCommand fails
 
+  // Change the position accuracy using UBX-CFG-NAV5
+  bool setNAV5PositionAccuracy(uint16_t meters = 100, uint16_t maxWait = defaultMaxWait);
```

Comment:
Please change to ```metres```

Reply:
Also, using 100m as the default is not a great idea. Please change to:

```
bool setNAV5PositionAccuracy(uint16_t metres, uint16_t maxWait = defaultMaxWait);
```

to force the user to define the pAcc.

Reply:
Thanks. I used to have a sodaq tracker that had this position accuracy build in, an default set to 100m, so that's why i added it here. Totally unrelated and updated in last push.